tests/unit/integration-test.js

```diff
@@ -1,3 +1,9 @@
+import Ember from 'ember';
```

Comment:
same here. Does the test build include the addon or is it because of the blueprint and bower install?